tools/mocha-chrome-wrapper

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+'use strict';
+const MochaChrome = require('mocha-chrome');
+const fs = require('fs');
+
+let promises = process.argv.slice(2).map(testScript => {
+  // we're not doing a ton of checking for now, since we've relied on
+  // shell globs to populate the parameter list with valid files in the only
+  // caller we care about for now (npm test).
+  let url = 'file://'+fs.realpathSync(testScript);
+
+  const options = Object.assign({
+    url: url
+  });
+
+  const mochaChrome = new MochaChrome(options);
+
+
+  let promise = new Promise(async (resolve) => {
+    // 'ended' seems like the preferred event, but let's catch everything
+    // that the mocha-chrome cli does.
+    mochaChrome.on('ended', stats => {
+      resolve(stats.failures);
+    });
+    mochaChrome.on('failure', message => {
+      resolve(1);
+    });
+
+    console.log('running tests at: '+options.url);
+    await mochaChrome.connect();
+    await mochaChrome.run();
+  });
+  return promise;
+});
+Promise.all(promises).then((values) => {
+  let exitCode = values.reduce(
+    (accumulator, currentValue) => accumulator|=currentValue
+  );
+  process.exit(exitCode);
```

Comment:
For future consideration -- we might want logging here re: the url associated with this promise, to more quickly allow someone reading to see at process exit which of the html files failed if it's a non-zero exit code, but can wait to see what we think.

Reply:
Yeah this is good - it'll be much faster to just print this at the bottom.